### ISSUE TITLE
Error when opening `ReGenesees.GUI()` from R in Manjaro Linux

### ISSUE THREAD

**Issue by josephuses** [DiegoZardetto/ReGenesees.GUI#1]
I came upon your package when looking for GUI for analyzing complex surveys in R. This is actually the first time I have heard of this package although I have used the `survey` package by Lumley a lot in the past.

Regarding the issue, I get an error when running `ReGenesees.GUI()`.

```
 Error in structure(.External(.C_dotTclObjv, objv), class = "tclObj") : 
  [tcl] couldn't open "/home/R_libs/ReGenesees.GUI/images//RG.logo10.gif": no such file or directory.
```
The issue seems to stem up from the two slashes appended after `img.path` in the function `img.path` of `ReGenesees.GUI.R`.

**Comment by DiegoZardetto** [comment]
Hi josephuses,
sorry for my late reply.

Thanks for signaling the issue.

I'll try to fix it by using an OS independent file path specification (R function file.path should work pretty well).

I'll let you know.

Best,
D.

**Comment by DiegoZardetto** [comment]
Hi josephuses,
I think the issue has been fixed and I've pushed to Github a new version of the package (ReGenesees.GUI 2.1).

In the NEWS file you'll read what follows:

------------
Changes in ReGenesees.GUI version 2.1

 - THIS IS A DEVELOPMENT VERSION AND IS NOT RELEASED YET.
 - File paths to images were not built in a OS independent way: this caused
   issues in e.g. Manjaro Linux. Fixed. Thanks to josephuses.
------------

Thanks again.
D.
